src/config/config.go

```diff
@@ -80,7 +80,16 @@ var (
 	// CommonOptions tracks user-defined values that apply across commands
 	CommonOptions types.ZarfCommonOptions
 
-	// CLIArch is the computer architecture of the device executing the CLI commands
+	// CreateOptions tracks the user-defined options used to create the package
+	CreateOptions types.ZarfCreateOptions
+
+	// DeployOptions tracks user-defined values for the active deployment
+	DeployOptions types.ZarfDeployOptions
+
+	// InitOptions tracks user-defined values for the active Zarf initialization.
+	InitOptions types.ZarfInitOptions
```

Comment:
It seems these are not used anywhere

Reply:
Thanks must have been leftover from inherited code.